OxfordBodleian/Juel-Jensen/BDLaethf29.xml

```diff
@@ -165,62 +513,345 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                            <p>Both codices share, beside the identical handwriting, particular
+                                scribal practices. They show similar elements for the visual
+                                organisation: an almost identical ornamental band on the incipit
+                                page of the first text (more elaborated in <ref type="mss"
+                                    corresp="BDLaethf29"/>; in <ref type="mss" corresp="ESqma005"/>
+                                other reserved spaces for ornamental bands do exist but were not
+                                filled) and a very similar way of rubricating the incipit pages of
+                                texts and their sections (at the top: two red lines followed by two
+                                black and another two red lines; at the bottom: two red lines and
+                                two black lines). The name of the patron or scribe or owner,
+                                originally written in red ink throughout both manuscripts, was
+                                deleted. A rather weak evidence in <ref type="mss"
+                                    corresp="ESqma005"/> (see, e.g. <locus target="#86v"/>) points
+                                to the name of <foreign xml:lang="gez">ገብረ፡ ማርያም</foreign> (in this
+                                case, this original name was replaced in <ref type="mss"
+                                    corresp="BDLaethf29"/> with one and the same name, see
+                                below).</p>
+                            <p>However, in addition to the attested similar scribal practices, both
+                                codices share also physical peculiarities. Both have a quite
+                                particular, slender book format (more pronounced in <ref type="mss"
+                                    corresp="ESqma005"/>). Each cover shows a fine and careful
+                                blind-tooled decoration. Both have threads inserted into folia for
+                                navigation through the multiple-text-manuscripts; <ref type="mss"
+                                    corresp="ESqma005"/> has also marks painted in red to indicate
+                                the incipit page of a text or of a section. Extremely striking are
+                                two labels of two textile companies, one is glued into the inner
+                                part of the front board in <ref type="mss" corresp="ESqma005"/> and
+                                the other is sewn onto <locus target="#2v"/> in <ref type="mss"
+                                    corresp="BDLaethf29"/> (see <ref target="#e3"/>). <ref
+                                    type="mss" corresp="ESqma005"/> shows the label of “W&amp;I
+                                Peel”, the (later) company of the successful textile industrialist
```

Comment:
I think this is hugely interesting, but I wonder whether this should go in an article or in the introduction to our volume?